pkg/cluster/px/px.go

```diff
@@ -1506,6 +1537,15 @@ func (ops *pxClusterOps) runPXNodeWiper(pwxHostPathRoot, wiperImage, wiperTag st
 		},
 	}
 
+	// add '--debug' on TRACE debug-level
+	if logrus.IsLevelEnabled(logrus.DebugLevel) {
```

Comment:
For the `if` statement, should that be `logrus.IsLevelEnabled(logrus.TraceLevel)`?

Reply:
Oh, thanks for pointing it out.
My original implementation was switching logging only on `--trace`, but it was getting a bit over-crowded ("too many log output"), so I moved it to `--debug` -- and forgot to update the comment.  :)

Reply:
Fixed in [439640c](https://github.com/portworx/talisman/pull/129/commits/439640c2d5ad43aa5f22259d8596421a5715e848)